internal/cluster/cmd/clusterd/main.go

```diff
@@ -164,9 +183,33 @@ func main() {
 	go s.Run()
 
 	nm := nodes.NewManager(dir, cm, s, nodes.WithSharedConsensus(true))
+	handler := api.Handler(cm, s, nm, log.Named("api"))
+
+	var mu sync.Mutex
+	var v1ContractAddresses FixtureV1ContractAddresses
+	var v2ContractAddresses FixtureV2ContractAddresses
+	mux := http.NewServeMux()
+	mux.HandleFunc("/fixtures/explored/v1", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
Do we really need these routes? The explorer should be aware of existing contracts itself right? 
If I understand correctly the whole idea of adding the contract creation to the startup of the explorer was to avoid having to add routes to form contracts within tests.

Reply:
They are not strictly necessary.  We can remove the endpoints, but then in the testing code they'll have to look a couple blocks back and find the contract IDs. I guess I will remove them since Nate seems to be in favor of that as well.